src/Resources/contao/languages/de/tl_leaflet_layer.php

```diff
@@ -12,19 +12,19 @@
-$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_use_limit']['0']        = 'Offset und Limit für die Auflistung verwenden.';
-$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_use_limit']['1']        = 'Auswählen, falls Sie die Anzahl anzuzeigender Datensätze begrenzen möchten. Diese Einstellung ist beispielsweise nützlich, wenn Sie nur die ersten 500 Datensätze anzeigen möchten, oder falls Sie alle Datensätze anzeigen und dabei die ersten 10 überspringen wollen. Eine voreingestellte Paginierung bleibt dabei erhalten.';
-
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel']['0']                      = 'MetaModel';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel']['1']                      = 'Wählen Sie aus, welches MetaModel für diese Auflistung benutzt werden soll.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_filtering']['0']            = 'Anzuwendende Filtereinstellungen';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_filtering']['1']            = 'Bitte wählen Sie aus welche Filtereinstellungen für die Listenausgabe benutzt werden sollen.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_filterparams']['0']         = 'Filtereinstellungen überschreiben';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_filterparams_use_get']['0'] = 'Den GET-Parameter benutzen';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_limit']['0']                = 'Maximale Anzahl an Datensätzen';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_limit']['1']                = 'Bitte gegen Sie die maximale Anzahl der Datensätze an. Geben Sie eine 0 ein um die Paginierung zu deaktivieren.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_offset']['0']               = 'Listen-Offset';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_offset']['1']               = 'Bitte geben Sie den Wert für den Offset an (beispielsweise 10 um die ersten 10 Items zu überspringen).';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_sortby']['0']               = 'Sortieren nach';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_sortby']['1']               = 'Bitte wählen Sie die Sortierreihenfolge aus.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_sortby_direction']['0']     = 'Nach Richtung sortieren';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_sortby_direction']['1']     = 'Nach aufsteigender oder absteigender Reihenfolge  sortieren.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_use_limit']['0']            = 'Offset und Limit für die Auflistung verwenden.';
+$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_use_limit']['1']            = 'Auswählen, falls Sie die Anzahl anzuzeigender Datensätze begrenzen möchten. Diese Einstellung ist beispielsweise nützlich, wenn Sie nur die ersten 500 Datensätze anzeigen möchten, oder falls Sie alle Datensätze anzeigen und dabei die ersten 10 überspringen wollen. Eine voreingestellte Paginierung bleibt dabei erhalten.';
```

Comment:
German translation are managed at transifex, to these changes will get lost.

Reply:
Can't find the project at transifex. If you add me, I will transfer the translations.

Reply:
@cliffparnitzky the resource is here https://www.transifex.com/netzmacht/contao-leaflet-maps/translate/#de/metamodels-tl_leaflet_layer

You should ony add the new `$GLOBALS['TL_LANG']['tl_leaflet_layer']['metamodel_filterparams_use_get']['0']` and add a blank line at least

delete the chenages at de

Reply:
Reverted the german language file and added a blank line at the end of the english one.